app/components/footer.hbs

```diff
@@ -1,4 +1,6 @@
-<p class="{{local-class 'postscript' from='css-modules-to-scoped-css/components/welcome-page-copy.css'}}">
-  To migrate from ember-css-module to ember-css-scoped, you will have to rename <code>local-class</code> to <code>class</code>, 
-  and a couple of other things describes in the README.
-</p>
\ No newline at end of file
+<div>
```

Comment:
A new `<div>` in the `footer.hbs` component is due to a behavior of ember-scoped-css. It turns out that when that div is absent, ember-scoped-css doesn't add an additional class name to the elements used as tag selectors in the CSS. 

Let's imagine that in `footer.css` we add the following crazy style:
```css
p {
  color: blueviolet;
}
```

The built CSS would be something like:
```css
/* css-modules-to-scoped-css/components/footer.css */
p.e05f79fac {
  color: blueviolet;
}
```

And for the style to apply, we expect ember-scoped-css to add the sha as a class name on the p element: 
```
<p class="e05f79fac"></p>
```

But ember-scoped-css adds this sha class name only if the paragraph is wrapped in a div.